lib/team_api/canonicalizer.rb

```diff
@@ -9,8 +9,12 @@ class Canonicalizer
     # Canonicalizes the order and names of certain fields within site_data.
     def self.canonicalize_data(site_data)
       sort_collections site_data
```

Comment:
that's interesting - can you explain why?

My reason for preferring parens is that it immediately tells me that a method is being called and args are being passed when I see parens. otherwise, it takes me a few seconds to understand exactly what is going on